pkg/operator/tpr.go

```diff
@@ -59,7 +59,7 @@ func (o *Operator) createTPR() error {
 		}
 	}
 
-	return err
```

Comment:
If the intention is to swallow all the other errors, then we should at least log them.